index.html

```diff
@@ -6,43 +6,178 @@
+            <span>Multi-Post Stories</span>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">css</div>
+              </li>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+          <div class="work-card">
+            <div class="title">Profesional Art Printing Data</div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+          <div class="work-card">
+            <div class="title">Profesional Art Printing Data</div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+          <div class="work-card">
+            <div class="title">Profesional Art Printing Data</div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="top">
+          <span>About me</span>
```

Comment:
- [ ] [REQUIRED] Kindly use any of the other five header tags - Please replace the span in this <span>About me</span> to a header tag, either of this h2, h3, h4 or h5. Please refer to my previous comment for the reason behind this suggestion.